ai_worker/src/parsers/image_vision.py

```
@@ -0,0 +1,277 @@
+        Args:
+            file_path: 이미지 파일 경로
+
+        Returns:
+            VisionAnalysis: 비전 분석 결과
+
+        Raises:
+            FileNotFoundError: 파일이 존재하지 않는 경우
+        """
+        # 파일 존재 확인
+        if not Path(file_path).exists():
+            raise FileNotFoundError(f"Image file not found: {file_path}")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)